Tests/Other/WindowMonitorTests.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  WindowMonitorTests.swift
+//  XestiMonitors-iOS
+//
+//  Created by Martin Mungai on 20/03/2018.
+//
+//  © 2018 J. G. Pusey (see LICENSE.md)
+
+import UIKit
+import XCTest
+@testable import XestiMonitors
+
+internal class WindowMonitorTests: XCTestCase {
+    
+    let notificationCenter = MockNotificationCenter()
+    let window = UIWindow()
+    
+    override func setUp() {
+        super.setUp()
+        
+        NotificationCenterInjector.inject = {
+            return self.notificationCenter
+        }
+    }
+    
+    func testMonitorDidBecomeVisible() {
```

Comment:
Please add some minimal spacing. (See `KeyboardMonitorTests` for an example.)

Reply:
Changes requested have been implemented. Not too sure how I'm supposed to merge into master; this is actually my first time dealing with forks